src/app/api/github/route.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { NextResponse } from 'next/server';
+
+export async function GET() {
+    try {
+        const response = await fetch('https://mine-treasure.com/api/github', {
+            next: { revalidate: 3600 },
+        });
```

Comment:
this needs to be updated to what the existing site uses, because this will break when we migrate this to the main site